packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -16,7 +16,6 @@ import {
 import { ModelRegistry, TupaiaDatabase } from '@tupaia/database';
 import { AccessPolicy } from '@tupaia/access-policy';
 import { UnauthenticatedError } from '@tupaia/utils';
-import winston from 'winston';
```

Comment:
I removed the only actual log in this file in a previous PR because it was spammy